ClubMembersSortedCF/src/members/v1/PersonBuilder.java

```diff
@@ -0,0 +1,42 @@
+package members.v1;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+public class PersonBuilder {
+    private Person person = new Person();
+
+    PersonBuilder surname(String surname) {
+        person.setSurname(surname);
+        return this;
+    }
+
+    PersonBuilder firstName(String firstName) {
+        person.setFirstName(firstName);
+        return this;
+    }
+
+    PersonBuilder birthDate(LocalDate birthDate) {
+        person.setDateOfBirth(birthDate);
+        return this;
+    }
+
+    PersonBuilder residence(int zipcode, String settlement, String address) throws InvalidZipCodeException {
```

Comment:
kind of nitpicking, but [ZIP code](https://en.wikipedia.org/wiki/ZIP_Code) is written in 2 words - so we need to match the parameter's name accordingly